web/src/actions/aria.js

```diff
@@ -1,5 +1,13 @@
+export const ARIA_QUEUE_ANNOUNCEMENT = 'ARIA_QUEUE_ANNOUNCEMENT';
 export const ARIA_ANNOUNCE_CHANGE = 'ARIA_ANNOUNCE_CHANGE';
 
+export const ariaQueueAnnouncement = change => dispatch => {
+    dispatch({
+        type: ARIA_QUEUE_ANNOUNCEMENT,
+        change
+    });
+}
```

Comment:
@mgwalker What do you think about something like this? This would let us queue whatever ARIA announcement we might need, and do the logic of converting the announcement in the component instead of in a bunch of separate actions. (Although now that I've written that out, it occurs to me that maybe separate actions is a more React-y way of doing this than using `componentDidUpdate`?)

Reply:
It feels a little odd to have an action that is basically a component telling Redux to remind the component to do something later.  My gut tells me that distinct actions would be cleaner. 😕

Reply:
That's a fair criticism; it's messier this way than I was anticipating it might be. It seems like it can be DRY or it can be clean but it can't be both, which kinda sucks. (Or, at least, I'm not clever enough to figure out how to get it to be both.) Thanks for indulging me!